src/app/components/FormComponent/Form.tsx

```diff
@@ -12,13 +14,30 @@ import { Checkbox } from '@/app/components/CheckboxComponent/Checkbox';
 import { FormInitialData } from '@/app/types/formInitialData';
 import { initialData } from '@/app/components/FormComponent/helper';
 import { Modal } from '@/app/components/ModalComponent/Modal';
+import { getContent } from '@/lib/api';
+
+type ContentState = {
+  getMoreInOurInfoPackage: string | null;
+  fillTheForm: string | null;
+  iAgreeToReceive: string | null;
+  fullName: string | null;
+  country: string | null;
+};
+
+const toStringOrNull = (value: any, fallback: string = ''): string | null => {
```

Comment:
Po co to jest

Reply:
W sumie super pytanie, podejrzewam ze po nic

Reply:
a jak tego nie mam to typy nie działają